{
  try {
    mMemoryPressureRouter.destroy(reactApplicationContextResult.get());
  }
 catch (  Exception e) {
    FLog.w(ReactConstants.TAG,"Caught exception after cancelling react context init",e);
  }
 finally {
    mReactContextInitAsyncTask=null;
  }
}
